bidsify/bidsify.py

```diff
@@ -129,11 +129,14 @@ def bidsify_workflow(dicom_dir, heuristics, subject, session=None, output_dir='.
             age = np.round(age.days / 365.25, 2)
         else:
             age = np.nan
-
-        new_participant = pd.DataFrame(columns=['age', 'sex', 'weight'],
-                                       data=[[age, data.PatientSex,
-                                              data.PatientWeight]])
-        participant_df = pd.concat([participant_df, new_participant], axis=1)
+        column_set = set(participant_df.columns)
+        # If participant_df already has age, sex, and weight, skip
+        if not column_set.issuperset({'age', 'sex', 'weight'}):
```

Comment:
what if there's partial overlap?

Reply:
also should the new participant _not_ be added if the larger df has all three columns?